contracts/axelar-gateway/src/testutils.rs

```diff
@@ -192,3 +193,13 @@ pub fn rotate_signers(env: &Env, contract_id: &Address, new_signers: TestSignerS
         (),
     );
 }
+
+pub fn expect_invalid_action_error<T>(call_result: Result<T, ContractError>) {
```

Comment:
This should exist in the `std` crate, and be made into a macro that avoids the duplication of specifying mock auth and function name for the args (e.g. a simple usage like `assert_invoke_auth!(client.rotate_signers(...))`

Reply:
make sense. updated.